src/ceph-daemon/ceph-daemon

```diff
@@ -1301,10 +1301,10 @@ def command_unit():
 ##################################
 
 def command_ls():
-    ls = list_daemons()
+    ls = list_daemons(detail=not args.no_detail)
```

Comment:
This feels awkward, perhaps invert the arg and set a bool default of True?

For example:
`ceph-daemon ls --detail=False`

Reply:
i think ``--no-detail`` is friendlier than ``--detail=False``